### PR TITLE
CRAYSAT-1922: Adding iSCSI examples to the sat bootprep

### PR DESCRIPTION
**Pull request by shivaprasad-metimath**
# Description
IM:CRAYSAT-1922
Reviewer:Ryan

Adding example of iSCSI variant for the session template creation. 

Relates to:
- https://github.com/Cray-HPE/sat/pull/265

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by vinanti-p — APPROVED** [review]

File: operations/system_admin_toolkit/usage/SAT_Bootprep.md [hunk 1]
```diff
@@ -401,6 +401,30 @@ session_templates:
...
+      example_boot_set:
+        arch: X86
+        kernel_parameters: ip=dhcp quiet
+        node_roles_groups:
+        - Compute
+        rootfs_provider: sbps
+        rootfs_provider_passthrough: sbps:v1:iqn.2023-06.csm.iscsi:_sbps-hsn._tcp.{{default.system-name}}.{{default.site-domain}}:300
+```
+
+Note: Make sure to define the values for `system-name`and `site-domain` in the `site-vars.yaml` file
```
Comment (annapoorna-s-alt) [inline operations/system_admin_toolkit/usage/SAT_Bootprep.md]:
```suggestion
(**Note:** Make sure to define the values for `system-name`and `site-domain` in the `site-vars.yaml` file.)
```

**Review by annapoorna-s-alt — APPROVED** [review]

File: operations/system_admin_toolkit/usage/SAT_Bootprep.md [hunk 2]
```diff
@@ -486,6 +510,7 @@ input file support rendering as a Jinja2 template and thus support variables:
   `session_templates` key:
     - `name`
     - `configuration`
+    - `bos_parameters.bootsets.rootfs_provider_passthrough`
```
Comment (haasken-hpe) [inline operations/system_admin_toolkit/usage/SAT_Bootprep.md]: The key is `boot_sets` not `bootsets`. Also there is technically another key, which is the name of the boot set between `boot_sets` and `root_fs_provider_passthrough`. Let's try this:

```suggestion
    - `bos_parameters.boot_sets.BOOT_SET_NAME.rootfs_provider_passthrough`
```

Hopefully that will make it clear to the reader.

File: operations/system_admin_toolkit/usage/SAT_Bootprep.md [hunk 3]
```diff
@@ -381,7 +381,7 @@ refer to [BOS Session Templates](../../boot_orchestration/Session_Templates.md).
 
 Here is an example of a BOS session template that refers to an existing IMS
 image by name and targets nodes with the role `Compute` and the architecture
-`X86` in HSM:
+`X86` in HSM for DVS:
```
Comment (haasken-hpe) [inline operations/system_admin_toolkit/usage/SAT_Bootprep.md]:
```suggestion
`X86` in HSM. This session template uses CPS as the `rootfs_provider`:
```

File: operations/system_admin_toolkit/usage/SAT_Bootprep.md [hunk 4]
```diff
@@ -401,6 +401,30 @@ session_templates:
         rootfs_provider_passthrough: dvs:api-gw-service-nmn.local:300:nmn0
 ```
 
+Here is an example of a BOS session template that refers to an existing IMS
+image by name and targets nodes with the role `Compute` and the architecture
+`X86` in HSM for iSCSI:
```
Comment (haasken-hpe) [inline operations/system_admin_toolkit/usage/SAT_Bootprep.md]:
```suggestion
`X86` in HSM. This session template uses SBPS as the `rootfs_provider`:
```

**Review by haasken-hpe — APPROVED** [review]
Please apply suggestions.

**Comment by mtupitsyn** [comment]
@shivaprasad-metimath @haasken-hpe Please resolve conversations to get this merged.

**Comment by haasken-hpe** [comment]
I applied the suggestions, and squashed while rebasing on the latest release/1.6 branch.

**Comment by shivaprasad-metimath** [comment]
> I applied the suggestions, and squashed while rebasing on the latest release/1.6 branch.

Thank you @haasken-hpe !!

**Pull request merged**